configuration/transitions.md

```diff
@@ -26,7 +26,7 @@ The following transitions are available and executed in order.
 | Key | Description |
 |---|---|
 | maintain_info_document | Records metadata about the document such as when it was replicated. Enabled by default. |
-| update_clinics | Update clinic data on new data records, use refid for clinic lookup otherwise phone number. |
+| update_clinics | Adds a contact's info to a new data record. This is used to attribute an incoming SMS message or report to the appropriate contact. The `rc_code` value on the contact is used to match to the value of the form field set as the `facility_reference` in the [JSON form definition](https://github.com/medic/medic-docs/blob/master/configuration/forms.md#json-forms). This matching is useful when reports are sent on behalf of a facility by unknown or various phone numbers. If no contact is found using the facility reference, a contact match is attempted using the sender's phone number. |
```

Comment:
> If no contact is found using the facility reference, a contact match is attempted using the sender's phone number.

The fallback to matching by sender phone happens if no facility reference is defined, rather than not matched to an existent contact. 
Also, when the contact is matched with `rc_code` and the sender phone is different from contact phone, the contact doc will be updated so `phone` equals sender's phone.
 (https://github.com/medic/medic/blob/7c0404d88d839503b855ad30c48fa701fee69416/sentinel/src/transitions/update_clinics.js#L20)